.prettierrc.yaml

```diff
@@ -0,0 +1 @@
+singleQuote: true
```

Comment:
Can we enforce this is the openapi GitHub Actions workflow?

Reply:
We could add [this](https://github.com/marketplace/actions/prettier-check) but there is no official Github Action for it.